docs/administration/garbage-collection/_index.md

```diff
@@ -48,9 +50,10 @@ Each time you run garbage collection on your Harbor instance, Harbor will track
   * **Job ID:** Unique numerical value assigned by Harbor when a run is initiated.
```

Comment:
Please change `Job ID` to `Task ID` and also apply this change to `purge history`